### PR TITLE
[FR] Expose code generated properties

### PR DESCRIPTION
**Pull request by maririos**
This PR exposes code generated properties (no design applied here) for:
- Appearance in FormLine https://github.com/Azure/azure-sdk-for-net/issues/16663
- BoundingBox for DataTable https://github.com/Azure/azure-sdk-for-net/issues/15872
- Content-type BMP https://github.com/Azure/azure-sdk-for-net/issues/15873
- Pages parameter for Recognize Content https://github.com/Azure/azure-sdk-for-net/issues/15874

It also changes selection mark state order after feedback from arch board.  https://github.com/Azure/azure-sdk-for-net/issues/16662

### CODE REVIEW

File: sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md [hunk 1]
```diff
@@ -10,13 +10,17 @@
...
 - Added ability to pass parameter `ModelName` to `StartTraining` methods.
 - Added the properties `ModelName` and `Properties` to types `CustomFormModel` and `CustomFormModelInfo`.
 - Added type `CustomFormModelProperties` that includes information like if a model is a composed model.
 - Added property `ModelId` to `CustomFormSubmodel` and `TrainingDocumentInfo`.
 - Added properties `ModelId` and `FormTypeConfidence` to `RecognizedForm`.
-- Added support to `StartRecognizeContent` to recognize selection marks such as check boxes and radio buttons.
+- Added property `Appearance` to `FormLine` to indicate the style of the extracted text. for example, "handwriting" or "other".
+- Added property `BoundingBox` to `FormTable`.
+- Added support for `ContentType` `image/bmp` in recognize content and prebuilt models.
+- Added property `Pages` to `RecognizeContentOptions` to specify the page numbers to analyze.
```
Comment (kristapratico) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md:23]: will `language` be added in another PR?

Comment (kristapratico) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md:23]: Also unrelated, but I just noticed on line 6 that this should say preview.2 now:

- It defaults to the latest supported API version, which currently is `2.1-preview.1`.

File: sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs [hunk 2]
```diff
@@ -2178,6 +2236,8 @@ private void ValidateFormPage(FormPage formPage, bool includeFieldElements, int
                 Assert.AreEqual(expectedPageNumber, table.PageNumber);
                 Assert.Greater(table.ColumnCount, 0);
                 Assert.Greater(table.RowCount, 0);
+                // Currently not true for CustomForms. Issue https://github.com/Azure/azure-sdk-for-net/issues/16827
```
Comment (kristapratico) [inline sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs:2239]: Just curious, for supervised or unsupervised is it not sent?

**Review by kristapratico — APPROVED** [review]

File: sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md (hunk 1 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md:23]: yes. I will added in a future PR

File: sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs (hunk 2 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs:2239]: From my tests, both come back without table bounding box

File: sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md (hunk 1 above)
Comment (maririos) [reply sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md:23]: ohh good catch! I will update
